docs/update_bios.rst

```diff
@@ -0,0 +1,52 @@
+BIOS Updates
+============
+
+Below are the steps for updating the BIOS on the app and mon servers. You can also follow these steps to update the BIOS on your computer, such as the Admin Workstation Computer.
+
+What you need
+~~~~~~~~~~~~~
+
+  #. A clean USB device to download the ``.bio`` file
+  #. An internet-connected workstation, such as the *Admin Workstation*
+  #. A UPS (uninterrupted power supply), such as a surge-protecting power supply with a backup battery
```

Comment:
This is probably overkill for the BIOS update process. It's pretty quick and the chance of a power failure is pretty slim unless you're a PG+E customer.

Reply:
Fair. I have updated to say 'recommended but not required' --is that okay, or do you want totally removed? (This is hopefully their regular setup anyway...) see 5079f5e82